consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/blockcreation/IbftMiningCoordinator.java

```diff
@@ -57,15 +75,46 @@ public IbftMiningCoordinator(
   }
```

Comment:
nit: does start/stop affect being able to listen to events? I.e. should this move into the start?
_Probably_ overkill - but the event is posted to the IbftEventQueue, which is not being emptied prior to "start"

Reply:
Yeah, makes sense to start / stop listening in start / stop methods.  Updated.